### PR TITLE
fix(ado-204739): fixed race condition

### PR DESCRIPTION
**Pull request by bchhun**
Fixed the race condition between the docfx menu and the wordpress menu.

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Documentation content changes

## What is the current behavior?

There's a double top menu that shows up in certain pages within the documentation section managed by `docfx`.

## What is the new behavior?

Fixed the race condition issue between the 2 XHR calls that is made to fetch the `docfx` & the `wordpress` menu.

Ultimately, we only want to keep the `wordpress` menu. 

The `docfx` menu will show up only if the `wordpress` api call fails.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Contains **NO** breaking changes
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable): https://dev.azure.com/nventive/UNO%20Marketing/_workitems/edit/204739

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#Documentation Uno
```
Comment (jeromelaban) [inline doc/README.md:1]:
```suggestion
# Documentation Uno

```

**Review by jeromelaban — APPROVED** [review]
Thanks ! There's just a simple markdown typo at the top with the header, make sure to squash your commits.

File: doc/README.md (hunk 1 above)
Comment (bchhun) [reply doc/README.md:1]: on it

**Comment by ajpinedam** [comment]
@bchhun The CI is complaining because of the commit message, specifically the scope. It seems it does not like this "ado-204739". I would try just with "ado" or without specifying any scope in case "ado" doesn't work.

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**